src/components/cursor.js

```diff
@@ -1,94 +1,162 @@
 var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
 var utils = require('../utils/');
 
+var EVENTS = {
+  CLICK: 'cursor-click',
```

Comment:
it should be automated. People will forget to prepend the component name to the emitted event